### PR TITLE
fix: Fix earned text color to disabled (which is still active for the farms that you earn token) after wallet disconnect

### LINKED ISSUE #668: Farm earned textcolor after wallet disconnect is in active color
**Bug Description**
Farm earned textcolor after wallet disconnect is in active color

**Steps to Reproduce**

1. Go to farms
2. Connect your wallet
3. Check your earnings
3. Disconnect from your wallet
4. The farms that you earned look like this with active '?' color

Screenshot 2021-03-16 at 09 58 29

**Expected Behavior**
Should be in disabled text color

### PR DESCRIPTION
**Pull request by memoyil**
Fixes #668

### CODE REVIEW

File: src/views/Farms/components/FarmTable/Earned.tsx [hunk 1]
```diff
@@ -15,9 +15,13 @@ const Amount = styled.span<{ earned: number }>`
 
 const Earned: React.FunctionComponent<EarnedProps> = ({ earnings }) => {
   const { account } = useWeb3React()
-  const displayBalance = earnings !== null && account ? earnings.toLocaleString() : '?'
+  let amountEarned = earnings
+  if (account == null) {
```
Comment (hachiojidev) [inline src/views/Farms/components/FarmTable/Earned.tsx:19]: `!account` is preferable

**Review by hachiojidev — CHANGES_REQUESTED** [review]

File: src/views/Farms/components/FarmTable/Earned.tsx (hunk 1 above)
Comment (RabbitDoge) [reply src/views/Farms/components/FarmTable/Earned.tsx:19]: I'm surprised eslint doesn't throw an error for `==`

Comment (memoyil) [reply src/views/Farms/components/FarmTable/Earned.tsx]: done

**Review by RabbitDoge — CHANGES_REQUESTED** [review]
the commit description doesn't match the pr content

File: src/views/Farms/components/FarmTable/Earned.tsx (hunk 1 above)
Comment (RabbitDoge) [reply src/views/Farms/components/FarmTable/Earned.tsx]: You can use a ternary with a const. As a rule of thumb, avoiding let is always a good idea.

Comment (memoyil) [reply src/views/Farms/components/FarmTable/Earned.tsx]: Fixed

**Comment by memoyil** [comment]
@RabbitDoge can you explain me more? What is not matching?

**Review by memoyil — COMMENTED** [review]
Change to ternary condition

**Review by hachiojidev — APPROVED** [review]

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**